nav2_system_tests/src/waypoint_follower/tester.py

```diff
@@ -208,7 +208,7 @@ def main(argv=sys.argv[1:]):
     time.sleep(2)
     test.cancel_goal()
 
-    # a failure case
+    # Set a waypoint outside the bounds of the map
```

Comment:
You need to get the error code and make sure its correct for the type of error this represents

Reply:
A added a test inside the error_codes to verify the piping.

Reply:
I don't see anywhere in that which replaces the full failure -> application like what the WPF or Nav2Pose test would provide. I really think we need to have a test here as a strict requirement.